extensions/schema-compare/src/models/schemaCompareOptionsModel.ts

```diff
@@ -0,0 +1,1381 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as loc from '../localizedConstants';
+import * as mssql from '../../../mssql/src/mssql';
+import { isNullOrUndefined } from 'util';
+
+export class SchemaCompareOptionsModel {
+	public deploymentOptions: mssql.DeploymentOptions;
+	public excludedObjectTypes: mssql.SchemaObjectType[] = [];
+
+	public optionsLookup = {};
```

Comment:
Write the type definition of these, right now they can be an object of basically anything

Reply:
will change this to a map in a separate PR